workflows/epitopeprediction.nf

```diff
@@ -113,6 +116,27 @@ workflow EPITOPEPREDICTION {
                     }
                 .set { ch_samples_from_sheet }
 
+    // gunzip variant files
```

Comment:
Maybe we can include another if statement within `subworkflows/local/input_check.nf` to first check for a .gz file and then just annotate that as `compressed`

Reply:
Yes I guess we can also put it in the `input_check` 👍. I would introduce `compressed_variants` since we might want to handle other gzipped types as well in the future.